src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -401,12 +408,15 @@ struct SvgLoaderData
     Array<SvgNode *> stack = {nullptr, 0, 0};
     SvgNode* doc = nullptr;
     SvgNode* def = nullptr;
+    SvgNode* cssStyle = nullptr;
```

Comment:
the parent node with styles - for now only one is allowed. for two <style> tags in one svg file the styles won;t be correctly applied. I can change it and use an array, but I don't know how to use title tag, to switch between styles...